django_project/kartoza_theme/templatetags/product.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def sort_option1(value):
```

Comment:
please change the name to indicate what purpose of the templatetags
maybe sort_product_options?